### ISSUE TITLE
Constants can't be override

### ISSUE THREAD

**Issue by NitroG42** [Kodein-Framework/Kodein-DI#125]
In kodein.kt:
```
        inner class ConstantBinder internal constructor(private val _tag: Any, private val _overrides: Boolean?) {
            /**
             * Binds the previously given tag to the given instance.
             *
             * @param T The type of value to bind.
             * @param value The instance to bind.
             * @param valueType The type to bind the instance to.
             * @throws OverridingException If this bindings overrides an existing binding and is not allowed to.
             */
            @Suppress("FunctionName")
            fun <T: Any> With(valueType: TypeToken<out T>, value: T) = Bind(tag = _tag) from InstanceBinding(valueType, value)
        }

```
here, the = Bind(tag = _tag) call lacks the apssing of the override parameters, should be 
= Bind(tag = _tag, _overrides)

**Comment by SalomonBrys** [comment]
Wooow...
This is an old one !
Thanks for the report !

This will be fixed in Kodein 5.0.1.
Will close this ticket when released.

**Comment by SalomonBrys** [comment]
Fixed in 5.0.1 ;)
